timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -55,16 +57,23 @@
 import com.palantir.lock.v2.StartTransactionResponseV4;
 import com.palantir.lock.v2.TimelockService;
 
+@RunWith(Parameterized.class)
 public class MultiNodePaxosTimeLockServerIntegrationTest {
     private static final String CLIENT_2 = "test2";
     private static final String CLIENT_3 = "test3";
     private static final List<String> ADDITIONAL_CLIENTS = ImmutableList.of(CLIENT_2, CLIENT_3);
 
-    private static final TestableTimelockCluster CLUSTER = new TestableTimelockCluster(
-            "https://localhost",
-            "paxosMultiServer0.yml",
-            "paxosMultiServer1.yml",
-            "paxosMultiServer2.yml");
+    @ClassRule
+    public static ParameterInjector<TestableTimelockCluster> injector =
+            new ParameterInjector<>(() -> PaxosSuite.NON_BATCHED_PAXOS);
```

Comment:
I think it would be good to have this go through a static-factory for purposes of naming, it wasn't obvious without looking through `ParameterInjector` that this is supposed to be the configuration provided if running outside a suite

Reply:
good shout